nixos/modules/config/xdg/portals/lxqt.nix

```diff
@@ -0,0 +1,50 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.xdg.portal.lxqt;
+
+in
+{
+  meta = {
+    maintainers = teams.lxqt.members;
+  };
+
+  options.xdg.portal.lxqt = {
+    enable = mkEnableOption ''
+      Desktop portal for LXQt desktops.
+
+      This will add the <package>lxqt.xdg-desktop-portal-lxqt</package>
+      package (with the extra Qt styles) into the
+      <option>xdg.portal.extraPortals</option> option.
```

Comment:
I don't think that will generate the right description.

Reply:
This is what I get on my notebook with `man configuration.nix`:

```man
       xdg.portal.lxqt.enable
           Whether to enable Desktop portal for LXQt desktops.

           This will add the lxqt.xdg-desktop-portal-lxqt package (with the extra Qt styles) into
           the xdg.portal.extraPortals option. .

           Type: boolean

           Default: false

           Example: true

           Declared by:
               <nixpkgs/nixos/modules/config/xdg/portals/lxqt.nix>
```

It looks good to me (maybe except for the uppercase **Desktop**).